ui/src/config.js

```diff
@@ -22,6 +22,7 @@ const defaultConfig = {
   defaultUIVolume: 100,
   enableUserEditing: true,
   enableSharing: true,
+  shareURL: 'http://127.0.0.1:4533',
```

Comment:
I think we should remove this, as it is not required for production, and could mess with running it in dev.